app/views/teams/_show_desktop.html.erb

```diff
@@ -28,13 +30,15 @@
     </div>
 
     <div class="flex justify-end pb-4">
-      <%= link_to "#", class: "text-primary box-shadow-medium flex items-center gap-1 rounded p-2 text-sm" do %>
-        <span class="text-primary text-sm">View full dashboard</span>
-        <span class="icon icon-arrow-right icon-small"></span>
+      <% policy(:dashboard).index? do %>
+        <%= link_to "#", class: "text-primary box-shadow-medium flex items-center gap-1 rounded p-2 text-sm" do %>
+          <span class="text-primary text-sm">View full dashboard</span>
+          <span class="icon icon-arrow-right icon-small"></span>
+        <% end %>
```

Comment:
Here also can create component and reuse in desktop and mobile view
also in show_desktop it is "View full dashboard" but in mobile view it is "View all metrics"

Reply:
ok I will add that